src/server.ts

```diff
@@ -362,6 +363,10 @@ const authV2Router = new AuthRouter({
   sgidAuthRouter,
 })
 
+const monitoringService = new MonitoringService({
+  launchesService,
+})
+
```

Comment:
could i check why's this initialised here and not the `support` container?

Reply:
hmm that would require creating launches service on the support container. no strong opinions on this, since this is a pretty light operation anyways

Reply:
to probe - when would this latency/cost matter? the support container is not user facing so latency doesn't really matter as opposed to creating it on the `app` container. (not that it's a big cost. i think the separation between user-facing and internal is more important here and what we want to maintain)

Reply:
not that it's a big cost. i think the separation between user-facing and internal is more important here and what we want to maintain)

so i am assuming we dont care about singleton pattern in our codebase ya? 
since in this case, launchesService will be created twice?

Reply:
i think this might be due to a mistake in rebase! **all** shared services should be exported from `"common/"`.

Reply:
ok, moved already